presto-spi/src/main/java/com/facebook/presto/spi/type/Decimals.java

```diff
@@ -248,6 +248,11 @@ public static void writeBigDecimal(DecimalType decimalType, BlockBuilder blockBu
         decimalType.writeSlice(blockBuilder, encodeScaledValue(value));
     }
 
+    public static void writeShortDecimal(BlockBuilder blockBuilder, long value)
+    {
+        blockBuilder.writeLong(value).closeEntry();
+    }
+
```

Comment:
this name is inconsistent with writeBigDecimal, the latter should be changed, but this is out of scope of this PR